projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.html

```diff
@@ -22,7 +22,6 @@
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
-
```

Comment:
Wanted to double check this change was intended?

Reply:
oh, it is not, probably a mis-enter when I look around.